### DISCUSSION TITLE
Crash on start right after installing from Discover on Steam Deck OLED

### DISCUSSION THREAD

**Discussion by steam3d** [ppy/osu#28304] [Q&A]
### Type

Crash to desktop

### Bug description

I installed 2024.521.2 version from Discover of Steam Deck OLED. Osu crash immediately crash after trying to run.

### Version

2024.521.2

### Logs

I can't run the osu to get the logs.

**Comment by BlockHead007** [comment]
Have you tried using AppImage or installing it using package manager?

**Comment by bdach** [comment]
For anyone else that might be as confused as me: "discover" appears to be referring to https://apps.kde.org/discover/ which is basically a frontend for flatpak. So the OP is installing a flatpak. The game isn't being featured on actual steam or anything.

That said I can't actually help diagnose this without any diagnostic info. We don't really support steam deck any more than any other linux computer. So I'm probably going to need like journalctl output (if that exists and is accessible on steam deck).
